### ISSUE TITLE
Popover box shadow conflicts with focus-visible 

### ISSUE THREAD

**Issue by anilanar** [chakra-ui/chakra-ui#6903]
### Description

When I open a popover, I expect it to have a shadow around it so that it creates an illusion of appearing on top of everthing else. But it doesn't have any box-shadow unless it's focused.

### Link to Reproduction

https://codesandbox.io/s/pensive-euclid-gbycbl

### Steps to reproduce

1. Go to [the sandbox](https://codesandbox.io/s/pensive-euclid-gbycbl?file=/src/index.js)
2. Click on "Click me"
3. See that there's no shadow around popover
4. Click on "Click me to see a shadow!"
5. See that now there's a shadow around popover!

### Chakra UI Version

2.3.6

### Browser

Firefox 101, Chromium 103

### Operating System

- [ ] macOS
- [ ] Windows
- [X] Linux

### Additional Information

Be aware that this is only reproducible when `focus-visible` polyfill is used.

**Comment by lexanth** [comment]
A workaround for this is to apply `data-focus-visible-disabled` to the `PopoverContent`, but agreed that this is a bug.

A similar bug occurs with `Menu` too.

**Comment by lukemcgregor** [comment]
@lexanth I have this issue on the menu popover, but applying `data-focus-visible-disabled` to the MenuList doesn't seem to help (I'm still losing the box shadow on mouse out of the menu). Is there another place that I should be putting it? It looks like this JS version isn't really required anymore as there is good browser support for focus-visible anyway.

**Comment by danielsarsi** [comment]
It seems this polyfill is selecting **all** focused elements.

**Comment by segunadebayo** [comment]
This can be resolved by removing the focus-visible polyfill and adding focus within styles

Something like: `_focusWithin={{ boxShadow: "outline" }}`

**Comment by ifightcrime** [comment]
On Chakra 2.7.0 @lexanth's fix of adding `data-focus-visible-disabled` worked, but @segunadebayo `_focusWithin={{ boxShadow: "outline" }}` did not.
